### PR TITLE
Input plugin(windows) for OpenHardwareMonitor application

### PR DESCRIPTION
**Pull request by marianob85**
- [x] README.md updated (if adding a new plugin)

This input plugin will gather sensors data provide by [OpenHardwareMonitor](http://openhardwaremonitor.org) application via Windows Management Instrumentation interface (WMI)

My second go program ( first was ping for windows ):
- WMI implementation from StackExchange was forked (https://github.com/marianob85/wmi) because float32 is not supported ( pull request is open )
- Upgrade and cleanup Godeps_windows ( to use with 'make windows' command )

Documentation of provided hardware and sensors: http://openhardwaremonitor.org/wordpress/wp-content/uploads/2011/04/OpenHardwareMonitor-WMI.pdf

### CODE REVIEW

**Comment by marianob85** [comment]
ci/circleci has a problem with dependencies for wmi which is only available for windows build

File: plugins/inputs/open_hardware_monitor/open_hardware_monitor.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
+// +build windows
```
Comment (sparrc) [inline plugins/inputs/open_hardware_monitor/open_hardware_monitor.go]: you need a newline after the build comment, like this:

```
// +build windows

package open_hardware_monitor
```

File: plugins/inputs/open_hardware_monitor/open_hardware_monitor_test.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// +build windows
```
Comment (sparrc) [inline plugins/inputs/open_hardware_monitor/open_hardware_monitor_test.go]: newline after build flag again

**Comment by marianob85** [comment]
Back to StackExchange/wmi ( remove marianob85/wmi deps ) - support for float32

**Comment by sparrc** [comment]
@marianob85 can you rebase and repush this?

**Comment by marianob85** [comment]
Done. Hops it's ok, because I'm not so familiar with version control

**Pull request closed without merging**